### DISCUSSION TITLE
faucet Link Token not sent to kovan network

### DISCUSSION THREAD

**Discussion by sako013** [smartcontractkit/full-blockchain-solidity-course-py#1480] [Q&A]
i have got the etheruem but never got the link and i am blocked now and not letting me more requests to the faucet
![faucetpng](https://user-images.githubusercontent.com/10506312/168419239-bd6487d1-5591-4dec-8aab-85792d7aac06.png)

**Comment by sako013** [comment]
i even tried to swap eth to link but did not work

**Comment by cromewar** [comment]
Have you added the token to metamask? remember you have to import the token to be listed on your metamask, it won't appear automatically, you must import it manually. 
Cheers.
